### ISSUE TITLE
Node Templates Problem: Dual Booting

### ISSUE THREAD

**Issue by WinstonWoof** [comfyanonymous/ComfyUI#1961]
I dual boot this PC but run ComfyUI from the same install folder in both boot environments and yet the saved Node Templates lists are different in each boot environment.
 
 Is this expected behaviour or is the data saved outside of the install folder somewhere?

**Comment by ltdrdata** [comment]
Node templates are stored in the browser cache. If you want to use them elsewhere, you need to export and import them elsewhere.

**Comment by WinstonWoof** [comment]
> Node templates are stored in the browser cache. If you want to use them elsewhere, you need to export and import them elsewhere.

cheers, seems an odd place to store them. What if you're the sort of person to configure the browser to clear the cache on exit?

You're screwed.

Hmm wonder what the rationale was behind that rather than saving to a config file inside the Install folder??

**Comment by senendds** [comment]
I also agree this is odd. I run comfyui in colab, through cloudflared, and each time I got a different url, so storing that in the browser cache is worse than useless...

**Comment by GalaxyTimeMachine** [comment]
Just discovered I've lost all of my node templates after clearing the web cache!! Please change this behaviour so it automatically saves them and makes them persistent.

**Comment by poisenbery** [comment]
> You're screwed.

Yes, indeed I am.

> browser cache

Why? :( All of my templates are gone because I was an idiot and assumed they were saved to a .json or something
